src/metric-engine/src/engine/create.rs

```diff
@@ -520,7 +532,8 @@ impl MetricEngineInner {
                 DATA_SCHEMA_TABLE_ID_COLUMN_NAME,
                 ConcreteDataType::uint32_datatype(),
                 false,
-            ),
+            )
+            .with_inverted_index(true),
```

Comment:
This is an incompatible change. Once the schema of a certain column is explicitly set to have an inverted key, the condition for indexing the tag by default will be destroyed.

Is it expected?

Reply:
oh is no problem for newly created tables.